Index: src/openvswitch.spec
===================================================================
--- src.orig/openvswitch.spec
+++ src/openvswitch.spec
@@ -127,7 +127,6 @@ License: ASL 2.0
 BuildArch: noarch
 Requires: python3
 Requires: python3-prometheus_client
-%{?python_provide:%python_provide python3-openvswitch = %{version}-%{release}}
 
 %description -n python3-openvswitch
 Python bindings for the Open vSwitch database
